lib/model/port.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * port.js
+ *
+ * Definition for the Port class. Ports are the points on a Node where
+ * connections attach to or from the node. They can be either inputs
+ * or outputs.
+ */
+
+/**
+ * Constructor function.
+ *
+ * @param id - The Port's unique identifier.
+ * @param type - The type of port. Valid options include "input" or "output".
+ */
+var Port = function(id,type) {
+	this.id = id || 0;
+	this.x = 0;

```

Comment:
Okay. I know that was the language spec but thought that it might have been converted.